File: briareos-master/dev/project/engine/modules/http_object.py
class HttpObject:
    def __init__(self, method, status_code, url, headers, body):
        self.method = method
        self.url = url
        self.headers = headers
        self.body = body

        if status_code is not None:
            self.statusCode = status_code
            self.type = "Response"
        else:
            self.type = "Request"

    def __str__(self):
        return "%s, %s, %s, %s" % (self.method, self.url, self.headers, self.body)

File: jbriareos-master/Tadeu_Freitas-jbriareos-c16b99c2c2d9/src/main/java/engine_modules_helpers/HttpObject.java
package engine_modules_helpers;

import java.util.Hashtable;

public class HttpObject {
    public String url;
    String method;
    Hashtable<String, String> headers;
    int statusCode;
    String objType;

    public HttpObject(String url, String method, Hashtable<String, String> headers, int statusCode) {
        this.url = url;
        this.method = method;
        this.headers = headers;
        this.statusCode = statusCode;
        this.objType = "Response";
    }

    public HttpObject(String url, String method, Hashtable<String, String> headers) {
        this.url = url;
        this.method = method;
        this.headers = headers;
        this.objType = "Request";
    }
}
